packages/shared/src/lib/tracking.ts

```diff
@@ -43,12 +51,13 @@ export const addTrackingParams = (baseUrl: string, params: Tracking): string =>
 export const addRegionIdAndTrackingParamsToSupportUrl = (
     baseUrl: string,
     tracking: Tracking,
+    numArticles?: number,
     countryCode?: string,
 ): string => {
     const isSupportUrl = /\bsupport\./.test(baseUrl);
 
     return isSupportUrl
-        ? addTrackingParams(addRegionIdToSupportUrl(baseUrl, countryCode), tracking)
+        ? addTrackingParams(addRegionIdToSupportUrl(baseUrl, countryCode), tracking, numArticles)
```

Comment:
I'm not sure why typescript isn't complaining about this? It seems here `numArticles` is of type `number | undefined` but in the signature of `addTrackingParams` it requires `number`

Reply:
Looking my VSCode seems to think it's just of type `number`
<img width="439" alt="Screenshot 2021-09-24 at 16 24 37" src="https://user-images.githubusercontent.com/17720442/134700643-31afeeb0-d86b-4101-bf16-fe40e9fc513e.png">

Reply:
It should be `number | undefined` because it's optional? Then should I make` numArticles` in `addTrackingParams` as optional too?

Reply:
Yeah I think it's the right move to mark it as optional in `addTrackingParams`, but I'm concerned as to why typescript is catching this atm

Reply:
I'll try and have a dig around

Reply:
ok thanks Tom!

Reply:
okay I can't figure out what the issue is here right now - but it's an existing thing so we shouldn't hold up this PR becasue of it. I'll create a ticket to fix this issue.